### DISCUSSION TITLE
Hope the sidebar of the website supports multiple languages.

### DISCUSSION THREAD

**Discussion by dufu1991** [biomejs/biome#991] [Ideas]
The sidebar in the astro.config of the current website seems to not support multilingual configuration, it is suggested to improve the structure to support multiple languages.

**Comment by ematipico** [comment]
I think it's a great idea.

Feel free to open a task (issue). FYI that's how the starlight website does it: https://github.com/withastro/starlight/blob/main/docs/astro.config.mjs#L69-L73

**Comment by morinokami** [comment]
@dufu1991 In https://github.com/biomejs/biome/pull/1085, I have translated the sidebar into Japanese and you can add the Chinese version as well in the same way! https://starlight.astro.build/reference/configuration/#sidebar should also be helpful.

**Comment by ematipico** [comment]
I think we can close the discussion :)
